kong/db/schema/init.lua

```diff
@@ -1359,59 +1384,82 @@ function Schema:process_auto_fields(input, context, nulls)
   end
   --]]
 
-  for key, field in self:each_field(input) do
+  if self.shorthands then
+    for _, shorthand in ipairs(self.shorthands) do
+      local sname = next(shorthand)
+      local sfunc = shorthand[sname]
```

Comment:
This should just be `local sname, sfunc = next(shorthand)`

Reply:
Fixed! I often forget about `next`'s second return!